### PR TITLE
Create break_continue_statement.c

### PR DESCRIPTION
**Pull request by Vaishnavi-iit**
Example of break and continue statement.

### CODE REVIEW

**Review by Anushka0912 — COMMENTED** [review]
Please use one more curly braces at the end of the program.

**Review by Anjali-1801 — CHANGES_REQUESTED** [review]
Just minor changes are required.

File: break_continue_statement.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        for(i=2;i<=x;i++)
+        {
+           if(x%i==0)
+           break;
+        }
+        if(i==x)
+        printf("%d\n",x);
+        else
+        continue;
+    }
```
[the remark this answers is not in the document]
Comment (Vaishnavi-iit) [reply break_continue_statement.c:18]: I have made the correction.

[the remark this answers is not in the document]
Comment (Anjali-1801) [reply break_continue_statement.c:18]: The program looks fine to me, I'm merging the PR.

**Pull request merged**